src/ui/title-ui-handler.ts

```diff
@@ -75,6 +78,56 @@ export default class TitleUiHandler extends OptionSelectUiHandler {
     this.appVersionText.setOrigin(0.5, 0.5);
     this.appVersionText.setAngle(0);
     this.titleContainer.add(this.appVersionText);
+
+    const currentLanguage = i18next.resolvedLanguage ?? "en";
+    const getTimeFormat = (): boolean => {
+      switch (currentLanguage) {
+        case "en":
+        case "es-ES":
+        case "ko":
+        case "zh-TW":
+        default:
+          return true; // 12h
+        case "de":
+        case "fr":
+        case "it":
+        case "ja":
+        case "pt-BR": // <-- in review
+        case "zh-CN":
+          return false; // 24h
+      }
+    };
+    const startDate = new Date(1738994400000);
+    const endDate = new Date(1739167200000);
+    const dateOptions: Intl.DateTimeFormatOptions = {
+      dateStyle: "short",
+      timeStyle: "short",
+      hour12: getTimeFormat(),
+    };
+    const startDateLocalized = new Intl.DateTimeFormat(currentLanguage, dateOptions).format(startDate);
+    const endDateLocalized = new Intl.DateTimeFormat(currentLanguage, dateOptions).format(endDate);
+    const localizedAnnouncementString: { [key: string]: string } = {
+      "en": ` - INFORMATION - \nServer maintenance is scheduled for the following period:\n${startDateLocalized} until ${endDateLocalized}\nEnd date and hour are an estimate.\nMaintenance may end at an earlier or later time.`,
+      "de": ` - INFORMATION - German translation goes here:\n${startDateLocalized} until ${endDateLocalized}`,
+      "es-ES": ` - INFORMACIÓN -\nUn mantenimiento del servidor está programado para el siguiente período:\nDesde el ${startDateLocalized} hasta el ${endDateLocalized}.\nLa fecha y hora de finalización son aproximadas.\nEl mantenimiento podría finalizar antes o extenderse más de lo previsto.`,
+      "fr": ` - INFORMATION - \nUne maintenance du serveur est prévue sur la période suivante :\nDu ${startDateLocalized} au ${endDateLocalized}\nL’heure de fin est une estimation et peut s’avérer plus en avance ou tardive qu’annoncé.`,
+      "it": ` - INFORMATION - Italian translation goes here:\n${startDateLocalized} until ${endDateLocalized}`,
+      "pt-BR": ` - INFORMATION - Portugese translation goes here:\n${startDateLocalized} until ${endDateLocalized}`,
+      "zh-TW": ` - 通知 - \n伺服器預計在以下時間維護：\n${startDateLocalized} 至 ${endDateLocalized}\n維護結束時間是預計時間\n維護可能稍早或稍晚結束。`,
+      "zh-CN": ` - 通知 - \n服务器预计在以下时间维护：\n${startDateLocalized} 至 ${endDateLocalized}\n维护结束时间是预计时间\n维护可能稍早或稍晚结束。`,
+      "ko": ` - INFORMATION - Korean translation goes here:\n${startDateLocalized} until ${endDateLocalized}`,
```

Comment:
```suggestion
      "ko": ` - 공지사항 -\n 아래 기간동안 점검 예정입니다. :\n${startDateLocalized} ~ ${endDateLocalized}\n 종료시각은 예상시간입니다.\n 점검은 예상했던 것보다 빠르게 혹은 늦게  끝날 수 있습니다.`,
```

here is the korean localization
(Korean uses both a 12hrsr and 24hrs time format. so there's no problem)